pino.js

```diff
@@ -170,7 +184,7 @@ function Pino (opts, stream) {
 }
 
 Pino.prototype = new EventEmitter()
-defineLevelsProperty(Pino.prototype)
+defineLevelsProperty(Pino.prototype, defaultLevelsValues)
```

Comment:
The rest of @pinojs/core may disagree with me, but I think the constructor is the one place Pino can afford a little time.